### PR TITLE
implemented pg_backend_pid scalar function

### PR DESCRIPTION
**Pull request by jodok**

### CODE REVIEW

File: sql/src/test/java/io/crate/operation/scalar/postgres/PgBackendPidFunctionTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ */
+
+package io.crate.operation.scalar.postgres;
+
+import io.crate.operation.scalar.AbstractScalarFunctionsTest;
+import org.junit.Test;
+
+import static io.crate.testing.SymbolMatchers.isLiteral;
+
+
```
Comment (matriv) [inline sql/src/test/java/io/crate/operation/scalar/postgres/PgBackendPidFunctionTest.java]: remove extra empty line

File: sql/src/main/java/io/crate/operation/scalar/postgres/PgBackendPidFunction.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import io.crate.operation.scalar.ScalarFunctionModule;
+import io.crate.types.DataTypes;
+
+import java.util.Collections;
+
+public class PgBackendPidFunction extends Scalar<Integer, Void> {
+
+    public static final String NAME = "pg_backend_pid";
+
+    protected static final FunctionInfo info = new FunctionInfo(
```
Comment (matriv) [inline sql/src/main/java/io/crate/operation/scalar/postgres/PgBackendPidFunction.java]: - can be final.
- `info` -> `INFO` (checkstyle for constants)

File: sql/src/main/java/io/crate/operation/scalar/postgres/PgBackendPidFunction.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        new FunctionIdent(NAME, Collections.emptyList()), DataTypes.INTEGER,
```
Comment (matriv) [inline sql/src/main/java/io/crate/operation/scalar/postgres/PgBackendPidFunction.java]: Please move each argument of FunctionInfo() constructor to separate line for code style consistency.
(See for example: https://github.com/crate/crate/blob/master/sql/src/main/java/io/crate/operation/scalar/timestamp/CurrentTimestampFunction.java#L50)

**Comment by matriv** [comment]
Igtm. (Please wait  for tests to run though ;) )

File: blackbox/docs/sql/scalar.txt [hunk 4]
```diff
@@ -1744,6 +1744,32 @@ Example::
     a special SQL syntax, meaning that they must be called without trailing
     parenthesis (``()``).
 
+``pg_backend_pid()``
+--------------------
+
+The ``pg_backend_pid()`` system information function is implemented for
+enhanced compatibility with PostgreSQL. CrateDB will always return `-1` as
```
Comment (chaudum) [inline blackbox/docs/sql/scalar.txt]: double backtick

File: blackbox/docs/sql/scalar.txt [hunk 5]
```diff
@@ -1744,6 +1744,32 @@ Example::
...
+there isn't a single process attached to one query. In contrast to
```
Comment (chaudum) [inline blackbox/docs/sql/scalar.txt]: ... in contrast to ...

File: blackbox/docs/sql/scalar.txt [hunk 6]
```diff
@@ -1744,6 +1744,32 @@ Example::
...
+PostgreSQL, where this represents the Process ID of the server process
```
Comment (chaudum) [inline blackbox/docs/sql/scalar.txt]: process ID

File: CHANGES.txt [hunk 7]
```diff
@@ -39,6 +39,8 @@ Changes
 - Functions which accept regular expression flags now throw an error when
   invalid flags are provided.
 
+- Added the pg_backend_pid() function for enhanced PostgreSQL compatibility
```
Comment (chaudum) [inline CHANGES.txt]: end sentence with period

Comment (chaudum) [inline CHANGES.txt]: optional: format `pg_backend_pid()` as inline code with double backtick

**Pull request merged**